build.gradle

```diff
@@ -20,5 +21,20 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven { url "https://www.jitpack.io" }
     }
 }
+
+ext {
+    // Sdk and tools
+    minSdkVersion = 11
+    targetSdkVersion = 25
```

Comment:
Does that means the application requires Android 7.1 ? That would be a big -1 for me.